templates/registration/logged_out.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <form action="{% url 'logout' %}" method="post">
```

Comment:
The form action is set to the 'logout' URL, which is unnecessary here since the user is already logged out. Consider removing the form and CSRF token, as they are not needed for displaying a logout success message.